config/jobs/kubernetes/release/release-config.yaml

```diff
@@ -46,3 +46,12 @@ presubmits:
           args:
             - test
             - ./...
+  - name: verify
```

Comment:
```suggestion
  - name: pull-release-verify
```

Reply:
Nothing to be sorry for. Thank you for catching that